CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # `chainweb-node` Changelog
 
+## 2.16 (2022-08-23)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2022-09-01T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will expire on **2022-12-01T00:00:00Z**.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+Changes:
```

Comment:
@larskuhtz suggested that we include this link here: https://github.com/kadena-io/chainweb-node/compare/2.15...master